pkg/helpers/aws.go

```diff
@@ -71,12 +72,12 @@ func AwsErrorCode(err error) string {
 	return ""
 }
 
-func CloudformationDescribe(cf cloudformationiface.CloudFormationAPI, stack string) (*cloudformation.Stack, error) {
+func CloudformationDescribe(stack string) (*cloudformation.Stack, error) {
```

Comment:
Same here, looks like `cf cloudformationiface.CloudFormationAPI` is the provider.Cloudformation struct. We can change the session there.
https://github.com/convox/rack/blob/aws-retryer-throttling/provider/kaws/system.go#L271
https://github.com/convox/rack/blob/master/provider/kaws/kaws.go#L159

It's a good practice to have a retry/throtling logic so it's ok that we configure for the other services too. (in the `initializeAwsServices` function)